server/lobbyconnection.py

```diff
@@ -197,7 +201,12 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
     async def command_coop_list(self, message):
         """ Request for coop map list"""
         async with self._db.acquire() as conn:
-            result = await conn.execute("SELECT name, description, filename, type, id FROM `coop_map`")
+            result = await conn.execute(
+                select([
+                    coop_map.c.name, coop_map.c.description, coop_map.c.filename,
+                    coop_map.c.type, coop_map.c.id
+                ])
```

Comment:
```suggestion
                select([coop_map])
```
It will change the order, but that doesn't matter since we're accessing the results by column name anyways.